modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/StoreTree.pm

```diff
@@ -691,7 +691,7 @@ sub call_hcs_all_trees {
 
 sub examl_exe_decision {
     my $self = shift;
-    my $avx = `grep avx /proc/cpuinfo`;
+    my $avx = $self->run_command(['grep', 'avx', '/proc/cpuinfo'])->out;
```

Comment:
Why not use `get_command_output` here?

Reply:
Because `get_command_output` sets `die_on_failure`, but `grep` returns 1 when it doesn't find any hit, making it look like a failure. I had two choices:
- `run_command(...)->out`
- `get_command_output(..., {die_on_failure => 0})`
I chose the former because it's shorter